packages/calcite-components/src/components/chip/chip.scss

```diff
@@ -10,7 +10,8 @@
  * @prop --calcite-chip-icon-color: Specifies the component's icon color.
  * @prop --calcite-chip-close-icon-color: Specifies the component's close element icon color.
  * @prop --calcite-chip-select-icon-color: Specifies the component's selection element icon color.
- * @prop --calcite-chip-select-icon-color-pressed: Specifies the component's selection element icon color when active.
+ * @prop --calcite-chip-select-icon-color-pressed: [deprecate] Use `--calcite-chip-select-icon-color-press`. Specifies the component's selection element icon color when active.
+ * @prop --calcite-chip-select-icon-color-press: Specifies the component's selection element icon color when active.
```

Comment:
@alisonailea it doesn't look like this new var is used anywhere? Should there also be tests here?